src/Centreon/Domain/Entity/Topology.php

```diff
@@ -304,6 +316,22 @@ public function setTopologyShow(string $topology_show): void
         $this->topology_show = $topology_show;
     }
 
+    /**
+     * @return string
+     */
+    public function getIsDeprecated(): ?string
```

Comment:
```suggestion
    public function IsDeprecated(): bool
```

Reply:
It is the legacy code. I can't change it cause it breaks entity creation